examples/work-instructions/main.js

```diff
@@ -0,0 +1,58 @@
+import { configureViewer } from '../helpers.js';
+import steps from './steps.js';
+import {
+  applyWorkInstruction,
+  applyCamera,
+  initializeWorkInstructions,
+} from './instructions.js';
+
+document.addEventListener('DOMContentLoaded', () => {
+  main();
+});
+
+async function main() {
+  const viewer = document.querySelector('vertex-viewer');
+  viewer.config = {
+    network: {
+      apiHost: 'https://api.dev.vertexvis.io',
+      renderingHost: 'wss://rendering.dev.vertexvis.io',
+    },
```

Comment:
Suspect you didn't intend to include these.

Reply:
I did not... somehow managed to add those back with the lint fixes unintentionally. I'll update and remove those